app/site/content-security-policy.txt

```diff
@@ -1,5 +1,5 @@
 default-src 'self';
-script-src 'self' 'unsafe-inline' https://code.jquery.com https://cdn.jsdelivr.net https://stackpath.bootstrapcdn.com https://f.ataylor.io https://polyfill.io https://cdn.jsdelivr.net https://giscus.app;
+script-src 'self' 'unsafe-eval' 'unsafe-inline' https://code.jquery.com https://cdn.jsdelivr.net https://stackpath.bootstrapcdn.com https://f.ataylor.io https://polyfill.io https://cdn.jsdelivr.net https://giscus.app;
```

Comment:
I've verified that adding `unsafe-eval` here resolves the issues outlined in #60 for Chrome, Edge, and Safari via local testing and in BrowserStack.

Can follow-up with a switch to `wasm-unsafe-eval` once that is available.